src/components/Todos/index.js

```diff
@@ -26,20 +50,56 @@ function amendStateWithChildren(DOMSource) {
   };
 }
 
+// THE MAIN FUNCTION
+// This is the implementation of the `main` function that gets passed to
+// Cycle's `run` function.
+// The sources that have been passed to `run` as well are immediately
+// deconstructed into the variables `DOM`, `hashchange`, `initialHash` & `storage`.
 function Todos({DOM, hashchange, initialHash, storage}) {
+  // THE LOCALSTORAGE STREAM
+  // Here we create a localStorage stream that only streams
+  // the first value received from localStorage because we use
+  // the localStorage to "rehydrate" the app whenever we launch or refresh it.
```

Comment:
"because" :+1:
"rehydrate" :-1: because rehydrate is a specific term used to refer to isomorphic rendering where the state is bundled, serialized, passed to the client, then "hydrated" to become live state. This here is just reading stored initial data.